cgroup1/utils.go

```diff
@@ -131,11 +131,18 @@ func hugePageSizes() ([]string, error) {
 }
 
 func readUint(path string) (uint64, error) {
-	v, err := os.ReadFile(path)
+	f, err := os.Open(path)
 	if err != nil {
 		return 0, err
 	}
-	return parseUint(strings.TrimSpace(string(v)), 10, 64)
+	defer f.Close()
+	b := make([]byte, 80)
```

Comment:
Actually one more comment. Perhaps we should put a note above this for the reasoning behind 80 e.g. arbitrary value above 64 to account for trailing space (most I remember seeing was just one space but 🤷‍♂️), should fit on stack, extra 16 from 64 shouldn't hurt anyone etc.

Reply:
PTAL, if the comment needs more modification. Can you add a suggestion/the entire wording; so that I can commit directly in.